com.nomachine.nxplayer.yml

```diff
@@ -0,0 +1,60 @@
+app-id: com.nomachine.nxplayer
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+separate-locales: false
+command: launch_nxplayer
+tags:
+  - proprietary
+
+finish-args:
+  - --device=dri
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-download
+  - --persist=.nx
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=x11
+
+modules:
+  - name: nomachine
+    buildsystem: simple
+    build-commands:
+      - install -D $FLATPAK_ID.png $FLATPAK_DEST/share/icons/hicolor/256x256/apps/$FLATPAK_ID.png
+      - install -D $FLATPAK_ID.desktop $FLATPAK_DEST/share/applications/$FLATPAK_ID.desktop
+      - install -D $FLATPAK_ID.metainfo.xml $FLATPAK_DEST/share/metainfo/$FLATPAK_ID.metainfo.xml
+      - install -D apply_extra $FLATPAK_DEST/bin/apply_extra
+      - install -D launch_nxplayer $FLATPAK_DEST/bin/launch_nxplayer
+    sources:
+      - type: extra-data
+        only-arches:
+          - x86_64
+        url: https://download.nomachine.com/download/8.11/Linux/nomachine-enterprise-client_8.11.3_4_x86_64.tar.gz
+        sha256: 506a35b1333543fd3b3609f75ac4dce5830a5a5f1eee74655e739b174c83ae34
+        filename: nomachine-enterprise-client.tar.gz
+        size: 44534549
+        # https://downloads.nomachine.com/download/?id=15
+      - type: extra-data
+        only-arches:
+          - aarch64
+        url: https://download.nomachine.com/download/8.11/Arm/nomachine-enterprise-client_8.11.3_3_aarch64.tar.gz
+        sha256: e87bcab44ac97f845d749c56f18332564c39e18ec3f87ff7e2743ec4f2be0c8b
+        filename: nomachine-enterprise-client.tar.gz
+        size: 40265918
+        # https://downloads.nomachine.com/download/?id=86&distro=ARM
+      - type: file
+        path: com.nomachine.nxplayer.png
+      - type: file
+        path: com.nomachine.nxplayer.desktop
+      - type: file
+        path: com.nomachine.nxplayer.metainfo.xml
+      - type: script
+        dest-filename: apply_extra
+        commands:
+          - tar -xf nomachine-enterprise-client.tar.gz --strip-components 0
+          - NX_INSTALL_PREFIX=/app/extra ./NX/nxclient --install fedora
```

Comment:
This part outputs a bunch of logs when installing the flatpak you should probably redirect to /dev/null

Reply:
Added redirection to /dev/null